dcs/triggers.py

```diff
@@ -29,37 +33,108 @@ def dict(self):
             "x": self.position.x,
             "y": self.position.y,
             "zoneId": self.id,
-            "radius": self.radius,
             "color": self.color,
             "properties": self.properties
         }
 
+
+class TriggerZoneCircular(TriggerZone):
+    def __init__(self, _id, position: mapping.Point, radius=1500, hidden=False, name="", color=None, properties=None):
+        super(TriggerZoneCircular, self).__init__(_id, position, hidden, name, color, properties)
+        self.type = TriggerZoneType.Circular
+        self.radius = radius
+
+    def dict(self):
+        d = super(TriggerZoneCircular, self).dict()
+        d["type"] = int(self.type)
+        d["radius"] = self.radius
+        return d
+
     def __repr__(self):
-        return "TriggerZone({id}, {x}, {y}, {r}, '{m}', '{n}', '{o}')".format(
+        return "TriggerZoneCircular({id}, {x}, {y}, {r}, '{m}', '{n}', '{o}')".format(
             id=self.id, x=self.position.x, y=self.position.y, r=self.radius, m=self.name, n=self.color, o=self.properties
         )
 
 
+# DCS mission format misspells the plural of "vertex". We follow this convention within PyDCS.
+class TriggerZoneQuadPoint(TriggerZone):
+    def __init__(self, _id, position: mapping.Point, verticies: List[mapping.Point],
```

Comment:
What is this position? I know that the miz needs this for serialization, but I don't understand if there are any restrictions on how I pick it.

Reply:
@bobmoretti, any idea?

Reply:
Ah, alright then. I'll push a change to document that behavior, and potentially just remove it from the API? It seems doubtful that anyone making these zones actually cares, so we could just place it arbitrarily at the midpoint and simplify the API.

Reply:
That's a good call. I guess I can't really think of a use case (since as soon as the user edits the zone in the ME the marker will be repositioned). But perhaps expose the `position` parameter optionally, defaulting to the centroid if unspecified?